Dockerfile

```diff
@@ -21,7 +24,18 @@ COPY --from=build-epicli-wheel /src/core/src/epicli/dist/ /epicli/
 
 RUN apt-get update \
     && apt-get install --no-install-recommends -y \
-        gcc libffi-dev make musl-dev openssh-client ruby-full sudo tar unzip vim \
+        curl gcc gnupg2 libffi-dev make musl-dev openssh-client ruby-full sudo tar unzip vim \
+\
+    && echo "Adding APT repositories ..." \
+    && curl -s https://baltocdn.com/helm/signing.asc | APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add - \
```

Comment:
This is some wierd url :) Why not just download the binaries (statically linked) from Helm releases on github https://github.com/helm/helm/releases/tag/v3.3.1 ? :)
https://www.getbalto.com/about.html this looks like a cheezy startup company to me :D